README.md

```diff
@@ -49,8 +47,10 @@ nuget_package(
 ```
 
 The `csharp_repositories` rule fetches external dependencies, namely the NUnit
-binaries, and the `csharp_configure` rule sets up the rules to use the local
-Mono C# toolchain installed on your system.
+binaries and the mono repository. Setting `use_local_mono` to True will use
+your installed mono framework instead of downloading one. If you are on OS X
+you can set use_local_mono to False and mono will be downloaded for you by
```

Comment:
Nit: back quotes around `True`, `False`, and `use_local_mono` elsewhere in the text.